### PR TITLE
[KBFS-2743] fs: Show team icons in browser when displaying teams

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: shared/fs/index.js [hunk 1]
```diff
@@ -66,10 +52,10 @@ const FileRow = RowConnector(({path, name, icon, onOpen}: FileRowProps) => (
 
 const rowPlaceholderIcon = isMobile ? 'icon-folder-private-24' : 'icon-folder-private-24'
 const placeholderTextStyle = {
-  width: '256px',
   backgroundColor: 'lightGrey',
   height: '16px',
   marginTop: '4px',
+  width: '256px',
```
Comment (chrisnojima) [inline shared/fs/index.js]: don't add px. its not compatible with native and react will add it for you automatically (except lineheight)

File: shared/constants/types/fs.js [hunk 2]
```diff
@@ -48,9 +48,8 @@ export const stringToPath = (s: string): Path => (s.indexOf('/') === 0 ? s : nul
 export const pathToString = (p: Path): string => (!p ? '' : p)
 export const getPathName = (p: Path): string => (!p ? '' : p.split('/').pop())
 export const getPathElements = (p: Path): Array<string> => (!p ? [] : p.split('/').slice(1))
-export const getPathVisibility = (p: Path): Visibility => {
-  if (!p) return null
-  const [, , visibility] = p.split('/')
+export const getVisibilityFromElems = (elems: Array<string>) => {
+  const [, visibility] = elems
```
Comment (buoyad) [inline shared/constants/types/fs.js]: :sheep: `const visibility = elems[1]` is much faster than this

Comment (jzila) [reply shared/constants/types/fs.js]: True but that also fails if `elems.length < 2`

Comment (chrisnojima) [reply shared/constants/types/fs.js]: what does that mean?

Comment (jzila) [reply shared/constants/types/fs.js]: It means that using `const visibility = elems[1]` precipitates the need for an `elems.length` check first. It's fine, I can do it, but I originally used this syntax based on feedback.

Comment (chrisnojima) [reply shared/constants/types/fs.js]: still confused.
```
const [, v] = [1]
```
turns into
```
var _ref = [1],
    v = _ref[1];
```

you don't need to check any lenght. its value is just undefined

**Comment by jzila** [comment]
@chrisnojima: ready for another look.

File: shared/constants/types/fs.js [hunk 3]
```diff
@@ -70,6 +69,10 @@ export const getPathVisibility = (p: Path): Visibility => {
       return null
   }
 }
+export const getPathVisibility = (p: Path): Visibility => {
```
Comment (songgao) [inline shared/constants/types/fs.js]: Seems like this isn't used anymore?

Comment (songgao) [reply shared/constants/types/fs.js]: Also I'm curious about the name "visibility". It seems like just TLF type to me but I must be missing something here.

**Pull request merged**